src/mlpack/core/math/random.hpp

```diff
@@ -66,6 +66,31 @@ inline double Random(const double lo, const double hi)
 }
 
 /**
+ * Generates a binomial random number in the specified range.
+ */
+inline double BinomialRandom(const int input)
```

Comment:
I guess the comment is invalid since `input` is equal to the success probability (and the mean). Moreover `input` is a real variable and belongs to `[0, 1]`.
Actually this is the Bernoulli distribution since the binomial distribution depends on the number of trials. And the name of the function should be similar to others (`RandNormal`, `RandInt`, etc).
So, I suggest `RandBernoulli`.